README.md

```diff
@@ -79,9 +79,17 @@ return {
 
 ## ⌨️ Keymaps
 
+We expose open, close and toggle to bind against, example alternatives below:
+
+#### Toggle
+```lua
+vim.keymap.set("n", "<leader>k", '<cmd>lua require("kubectl").toggle()<cr>', { noremap = true, silent = true })
+```
+
+#### Open with ```<leader>k``` and close with ```q```
 ```lua
--- Recommended is to have the same open mapping as your close (```<leader>k```) the plugin for a toggle effect.
 vim.keymap.set("n", "<leader>k", '<cmd>lua require("kubectl").open()<cr>', { noremap = true, silent = true })
+vim.keymap.set("n", "q", '<cmd>lua require("kubectl").close()<cr>', { noremap = true, silent = true })
```

Comment:
i think it's better to give an example only for the `toggle()` case since it's most common.
Users can choose when and if they ever need the `open()` and `close()` but nonetheless have the option to use it.

Reply:
That's fair, changed it! Thanks for the review 🙏